Sources/Puppy/FileRotationLogger.swift

```diff
@@ -37,7 +37,10 @@ public class FileRotationLogger: FileLogger {
                 let oldArchivedFileURLs = ascArchivedFileURLs(fileURL)
                 for (index, oldArchivedFileURL) in oldArchivedFileURLs.enumerated() {
                     let generationNumber = oldArchivedFileURLs.count + 1 - index
-                    let rotatedFileURL = oldArchivedFileURL.deletingPathExtension().appendingPathExtension("\(generationNumber)")
+                    let rotatedFileURL = oldArchivedFileURL
+                        .deletingPathExtension()
+                        .appendingPathExtension("\(generationNumber)")
+                        .appendingPathComponent(".txt")
```

Comment:
Thanks for your proposal, but I intentionally use only numeric suffixes here in order to match the general log rotation format on Linux OS(without txt extension)...

Reply:
I apologize that this was in here, it wasn't intended to be here as instead work on my own fork. Should've done this spelling fix on a branch instead of my `main`!

However I do notice this is Linux's version of file rotation, however I think it may be more beneficial to (at least allow) not follow that convention. The use of the original file declaration allows developers to use names like `log.txt/log.log` and this makes viewing logs on mobile in the `Files` app possible. With the Linux convention, it does not.

Just something that I noticed as I would need to view logs on device instead of sharing them to a computer for viewing.

Reply:
At the same time, I don't think file rotation based on file size is either unfinished (work in progress) or not working as intended. It will only work on the first time that a max file size was set, and then the original log file can grow without bound.

Reply:
> Just something that I noticed as I would need to view logs on device instead of sharing them to a computer for viewing.

Thank you for sharing your idea.
I agree that we should add a new option to keep the file extension(e.g. log, txt) in order to give more options for users. I may add this option myself when I spare time🙂
Before that, I will merge the PR to fix typo!

> At the same time, I think file rotation based on file size is either unfinished (work in progress) or not working as intended.

IMO, I assume it depends on the change to keeping file extension...
At least, we need to modify more to rotate the file correctly while keeping the file extension.